google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/JobStatistics.java

```diff
@@ -320,8 +320,16 @@ static LoadStatistics fromPb(com.google.api.services.bigquery.model.JobStatistic
 
     private final Integer billingTier;
     private final Boolean cacheHit;
+    private final String ddlOperationPerformed;
+    private final TableId ddlTargetTable;
+    private final Long estimatedBytesProcessed;
+    private final Long numDmlAffectedRows;
+    private final List<TableId> referencedTables;
```

Comment:
I just realized: Do we want to make these immutable?

Reply:
Since this is job stats, which are read-only properties, I think immutable makes sense.